app/storage/secureStorage.ts

```diff
@@ -130,7 +153,7 @@ export async function encryptAndStoreAppKeyWithBiometrics(passcode: string) {
     }
 
     // Handle iOS
-    if (Platform.OS === 'ios') {
+    if (Platform.OS === 'ios' || Platform.OS === 'android') {
```

Comment:
Remove if?

Reply:
other platforms are in question here, but you are right if we build only for iOS and android this if statement is pointless